docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:9.5-alpine
```

Comment:
Is this the same version number we expect to use from software collections?

Reply:
The image should be https://hub.docker.com/r/centos/postgresql-96-centos7/ or registry.access.redhat.com/rhscl/postgresql-96-rhel7. The alpine image is postgres' own official image.

Reply:
Using the same version makes sense i.e. 9.6. centos for upstream sounds reasonable.  Is it a drop in replacement?

Reply:
Good point, going to try shifting the CI postgres image to this one too.

Reply:
Also timescaledb requires version 9.6 so if we are going to use that in the future I think it makes sense to use 9.6

Reply:
Looking at the metrics-apb PR we should change this to `registry.access.redhat.com/rhscl/postgresql-96-rhel7:latest` @paolobueno can you update it?